Add nuget package

riha opened the pull request:
Added NuGet package, see what you think

riha committed:
Ignoring ReSharper related files

riha committed:
Updating NuGet.exe

Updating NuGet.exe from 1.6.21205.9031 to 2.1.0

riha committed:
Adding info to AssemblyInfo

As we're basing the nuspec on the AssemblyInfo extra information is
needed

riha committed:
Adding a NuGet package specification file

riha committed:
Ignoring nuget packages

niik commented:
This is great. Let's figure out what to put in AssemblyCompany and I'll accept it. :+1:

riha committed:
Hardcoding authors in nuspec

Removing AssemblyCompany from AssemblyInfo and hardcoding authors and
owners directly in nuspec.

riha committed:
Updating to version 0.2.1

Need to bump version to be able to publish new nuget package

niik commented:
Thanks!
